azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/java/com/microsoft/azure/toolkit/lib/mysql/service/MySqlManagerFactory.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+package com.microsoft.azure.toolkit.lib.mysql.service;
+
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.resourcemanager.mysql.MySqlManager;
+import com.microsoft.azure.toolkit.lib.Azure;
+import com.microsoft.azure.toolkit.lib.AzureConfiguration;
+import com.microsoft.azure.toolkit.lib.auth.Account;
+import com.microsoft.azure.toolkit.lib.auth.AzureAccount;
+import com.microsoft.azure.toolkit.lib.common.cache.Cacheable;
+
+import java.util.Optional;
+
+public final class MySqlManagerFactory {
+
+    private MySqlManagerFactory() {
+    }
+
+    @Cacheable(cacheName = "MySqlManager", key = "$subscriptionId")
+    public static MySqlManager create(String subscriptionId) {
+        final Account account = Azure.az(AzureAccount.class).account();
+        final AzureConfiguration config = Azure.az().config();
+        final String userAgent = config.getUserAgent();
+        final HttpLogDetailLevel logLevel = Optional.ofNullable(config.getLogLevel()).map(HttpLogDetailLevel::valueOf).orElse(HttpLogDetailLevel.NONE);
+        final AzureProfile azureProfile = new AzureProfile(null, subscriptionId, account.getEnvironment());
+        return MySqlManager.configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(logLevel))
+            .withPolicy(getUserAgentPolicy(userAgent))
```

Comment:
may use `ProviderRegistrationPolicy` to register service provider automatically, refers https://github.com/microsoft/azure-maven-plugins/blob/develop/azure-toolkit-libs/azure-toolkit-applicationinsights-lib/src/main/java/com/microsoft/azure/toolkit/lib/applicationinsights/ApplicationInsights.java#L98

Reply:
will fix it in future PR